sass/px-modal.scss

```diff
@@ -93,16 +65,23 @@ $inuit-enable-btn--disabled        : true;
   }
 }
 
-.modal__buttons {
+.modal__header {}
+.modal__header__text {
+  margin-top: 0;
+  margin-bottom: $inuit-base-spacing-unit--large;
+}
+
+.modal__body {}
+.modal__body__text {
+  margin-top: 0;
   margin-top: $inuit-base-spacing-unit--large;
-  border-top: 1px solid var(--px-modal-border-color, transparent);
+  margin-bottom: $inuit-base-spacing-unit--large;
 }
-.modal__buttons button.btn {
-  &:nth-last-of-type(2) {
-    margin-right: $inuit-base-spacing-unit--small;
-  }
-  @apply --px-modal-buttons;
+
+.modal__triggers {
+  display: flex;
+  justify-content: flex-end;
 }
-.modal__buttons button.btn--call-to-action {
-  @apply --px-modal-positive-button;
+.modal__triggers__reject {
```

Comment:
Should be `.modal__triggers--reject` sense reject is a modifier of triggers.

Reply:
`.modal__triggers` is a parent container, `.modal__triggers__reject` is a child element of that container. So it wouldn't make sense to have it be a modifier right?

Reply:
according to this article (https://www.smashingmagazine.com/2016/06/battling-bem-extended-edition-common-problems-and-how-to-avoid-them/#1-what-to-do-about-grandchild-selectors-and-beyond)
It would be `modal__reject`